frontend/app/src/screens/diff/data-diff-node.tsx

```diff
@@ -201,34 +163,24 @@ export const DataDiffNode = (props: tDataDiffNodeProps) => {
       {commentsCount && (
         <div className="flex items-center" data-cy="comments-count" data-testid="comments-count">
           <Tooltip enabled content={"Total number of comments"}>
-            <div className="flex">
-              <ChatBubbleLeftRightIcon className="w-4 h-4 mr-2" />
-              <Pill className="mr-2">{JSON.stringify(commentsCount)}</Pill>
+            <div>
+              <Badge variant={"dark-gray"} className="rounded-full mr-2">
+                <Icon icon="mdi:message-fast-outline" className="mr-1" />
+                {commentsCount}
+              </Badge>
             </div>
           </Tooltip>
         </div>
       )}
 
-      <div className="flex items-center mt-2 lg:mt-0">
-        <DiffPill {...summary} />
-
-        <div className="flex lg:w-[200px]">
-          {changed_at && <DateDisplay date={changed_at} hideDefault />}
-        </div>
-      </div>
-
       {!branchName && <DataDiffConflictInfo path={path} />}
```

Comment:
<img width="899" alt="Screenshot 2024-08-01 at 15 51 20" src="https://github.com/user-attachments/assets/fb026433-2a99-4973-adcb-15550e9a4d1b">
 When I click on (i) in the blue side, an empty gray bar appears